scripts/mkcloud

```diff
@@ -1431,11 +1431,19 @@ function expand_steps()
             if [ $onecmd = $cmd ] ; then
                 found=1
                 case "$cmd" in
+                    setupcompute|instcompute)
+                        corrected=${cmd/compute/nodes}
+                        echo "WARNING: the '$cmd' step is deprecated; use '$corrected' instead." >&2
```

Comment:
That would break Jenkins and maybe other people's scripts - I assumed it would make people unhappy, but if you think it's OK then I can certainly do it.